config/serverless.oblt.essentials.yml

```diff
@@ -2,4 +2,18 @@
 
 ## Disable xpack plugins
 xpack.infra.enabled: false
-xpack.slo.enabled: false
\ No newline at end of file
+xpack.slo.enabled: false
+xpack.apm.enabled: false
+uiSettings.overrides.defaultRoute: /app/discover
+
+xpack.legacy_uptime.enabled: false
+xpack.ux.enabled: false
+xpack.uptime.enabled: false
+xpack.observabilityAIAssistant.enabled: false
+
+xpack.fleet.internal.registry.excludePackages: [
+  # Oblt integrations
+  'synthetics',
+  'uptime',
+  'synthetics_dashboards',
+]
```

Comment:
Could you confirm if the configuration array values are merged correctly? Or is this completely overriding the longer list of excluded packages defined in `serverless.oblt.yml`?

Reply:
looks like they were not, good catch. I've duplicated the list from the root yaml, and made additions.